index.html

```diff
@@ -0,0 +1,211 @@
+                        PHXDW,
+                        SustainbleUX, Audubon society, and Target HQ. My work has been published in the Cooper Hewitt’s
+                        Summer
+                        2019 Design Journal among other peer reviewed journals.
+                    </p>
+                </li>
+                <li class="speaker-info">
+                    <p class="title">
+                        Collaborating With Community
+                    </p>
+                    <p class="time">2PM - 2:45PM</p>
+                    <img src="https://cdn.glitch.com/74171c12-683c-4072-8113-bc7e1b6a35cc%2FSuhailaSikand%20copy.jpg?v=1607029785493"
+                        alt="smiling close-up of a dark haired woman with outdoor plants behind her"
+                        class="speaker-photo" />
+                    <a class="speaker-name" href="http://www.suhaila.me/" target="blank">Suhaila Sikand</a>
+                    <p class="bio">Climate designer, exploring community, equity, behavior, & design</p>
+                    <p>
+                        As a social impact and climate designer, I am grounded in community, equity, and sustainability.
+                        Unfortunately, human-centered design doesn’t always support these principles with the right
+                        intentions.
+                        Instead of seeing a problem in a community, let’s get invited to work with the community.
+                        Instead of
+                        interpreting people’s needs, let’s include them in the process. <strong>Human centered design
+                            implies a
+                            surface
+                            level engagement where the power of design remains with the designer, not the
+                            community.</strong>
+                    </p>
+                </li>
+                <p class="previous-speakers"> <a href="#">View previous speaker events </a></p>
+            </ul>
+
+        </section>
+
+        <section id="register">
+            <h2><span aria-hidden="true">-</span> register <span aria-hidden="true">-</span></h2>
+
+            <form class="registration">
+                <ul class="u-list-reset">
+                    <li>
+                        <label for="name">name:</label>
+                        <input id="name" type="text" name="user_name">
+                    </li>
+                    <li>
+                        <label for="mail">email:</label>
+                        <input id="mail" type="email" name="user_email">
+                    </li>
+                    <li>
+                        <label for="phone-number">phone:</label>
+                        <input type="phone" name="phone" </li> <li>
+                    </li>
+                    <li class="button">
+                        <button class="register-button" class="button-reset" type="submit">Register Now</button>
```

Comment:
You want to combine your `class` values on this element:
```suggestion
                        <button class="register-button button-reset" type="submit">Register Now</button>
```